src/helpers/serializer.js

```diff
@@ -0,0 +1,15 @@
+export const deserializeItems = (todos) => {
+  return todos.map((todo) => {
+    return deserializeItem(todo);
+  });
+};
+
+export const deserializeItem = (todo) => {
+  return { ...todo, id: getIdByUrl(todo.url) };
```

Comment:
In a serializer I suggest mapping all of the props to their corresponding keys.

Even in scenarios where all of the props names match their key names 
it is helpful for a matter of scalability considering the API might change in the future.